linkerd/app/inbound/src/lib.rs

```diff
@@ -160,26 +174,26 @@ pub fn spawn<P>(
             |req: &http::Request<_>| {
                 let dst = req
                     .headers()
-                    .get(super::CANONICAL_DST_HEADER)
+                    .get(CANONICAL_DST_HEADER)
                     .and_then(|dst| {
                         dst.to_str().ok().and_then(|d| {
                             Addr::from_str(d).ok().map(|a| {
-                                debug!("using {}", super::CANONICAL_DST_HEADER);
+                                debug!("using {}", CANONICAL_DST_HEADER);
                                 a
                             })
                         })
                     })
                     .or_else(|| {
-                        super::http_request_l5d_override_dst_addr(req)
+                        http_request_l5d_override_dst_addr(req)
                             .ok()
                             .map(|override_addr| {
-                                debug!("using {}", super::DST_OVERRIDE_HEADER);
+                                debug!("using l5d-dst-override");
```

Comment:
Is there a reason this no longer uses the constant?

Reply:
no, got lost in the merge